frontend/src/main/scala/bloop/testing/TestInternals.scala

```diff
@@ -37,6 +39,37 @@ object TestInternals {
     new FilteredLoader(getClass.getClassLoader, filter)
   }
 
+  /**
+   * Parses `filters` to produce a filtering function for the tests.
+   * Only the tests accepted by this filter will be run.
+   *
+   * `*` is interpreter as wildcard. Each filter can start with `-`, in which case it means
+   * that it is an exclusion filter.
+   *
+   * @param filters A list of strings, representing inclusion or exclusion patterns
+   * @return A function that determines whether a test should be run given its FQCN.
+   */
+  def parseFilters(filters: List[String]): String => Boolean = {
+    val (exclusionFilters, inclusionFilters) = filters.partition(_.startsWith("-"))
+    val inc = inclusionFilters.map(toPattern)
+    val exc = exclusionFilters.map(f => toPattern(f.tail))
+
+    (inc, exc) match {
+      case (Nil, Nil) =>
+        _ =>
```

Comment:
Yak-shaving: can we use parenthesis for these functions and make them one-liners to make them easier to read?